### PR TITLE
feature/squad creator component

### PR DESCRIPTION
**Pull request by TanyaAkulich**

### CODE REVIEW

File: src/components/NewSquad.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    <Container>
+      <Paper className={classes.newSquadMessageRoot} variant="outlined" square>
+        <div className='d-flex flex-column flex-sm-row'>
+          <SVG className={classes.newSquadMessageIcon} src='icons/soldier.svg' width='50%'/>
+          <div className='d-flex flex-column ml-sm-4 mt-4 mt-sm-0'>
+            <Typography className={classes.newSquadMessageTitle} variant='h4' component='h1'>
+              Похоже что вы ещё не вступили ни в один взвод...
+            </Typography>
+            <Typography className={classes.newSquadMessageText} variant='h6' component='h2'>
+              Отправляйте запросы на вступление и ожидайте подтверждения командира взвода. Мы уведомим вас как только ваша заявка будет подтверждена.
```
Comment (ARtoriouSs) [inline src/components/NewSquad.jsx:43]: I think it's better to use
`Отправьте запрос на вступление ...`
because user can only send one request

**Review by ARtoriouSs — CHANGES_REQUESTED** [review]
Looks good!

**Comment by ARtoriouSs** [comment]
Everything is fine, lets just add validations for input

File: src/components/NewSquad.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+export default function NewSquad() {
+  const classes = useStyles();
+  const user = useSelector(state => state.currentUser)
+  const [day, setDay] = React.useState(0);
+  const [squadNumber, setSquadNumber] = React.useState();
+  const [sendRequest] = useMutation(SET_SQUAD, { onCompleted: () => window.location.href ='/my-squad' });
+
+  if (user && user.squad) {
+    return window.location.href ='/my-squad'
+  }
```
Comment (str1nker) [inline src/components/NewSquad.jsx]: please do not use `window.location.reload` . As I remember I changed that in my commit here, incorrect conflict resolution?

**Review by str1nker — CHANGES_REQUESTED** [review]

File: src/components/NewSquad.jsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+
+const useStyles = makeStyles(SquadsStyles);
+
+export default function NewSquad() {
+  const classes = useStyles();
+  const user = useSelector(state => state.currentUser)
+  const [day, setDay] = React.useState(0);
+  const [squadNumber, setSquadNumber] = React.useState();
+  const [sendRequest] = useMutation(SET_SQUAD, { onCompleted: () => window.location.href ='/my-squad' });
```
Comment (str1nker) [inline src/components/NewSquad.jsx]: Need to fix this

**Review by str1nker — CHANGES_REQUESTED** [review]

**Pull request merged**